.changeset/shaggy-crabs-thank.md

```diff
@@ -0,0 +1,9 @@
+---
+"jspsych": major
+"@jspsych/plugin-audio-button-response": minor
+"@jspsych/plugin-audio-keyboard-response": minor
+"@jspsych/plugin-audio-slider-response": minor
+"@jspsych/test-utils": minor
+---
+
+Changed plugins to use AudioPlayer class; added tests using AudioPlayer mock; plugins now use AudioPlayerInterface; changed test-utils to fix clickTarget
```

Comment:
Since this will show up in the changelog, it might make sense to split into two changeset files. One for audio stuff, the other for test utils

Reply:
Changeset split.